### DISCUSSION TITLE
Question about the variables ("e_form_2000" and "e_form_500") in "analyze_chgnet.py" file

### DISCUSSION THREAD

**Discussion by Claudia-Hello** [janosh/matbench-discovery#80] [Q&A] [answered]
Hi,
I got a problem when I ran the `analyze_chgnet.py` file. It seem that there are no columns named `e_form_per_atom_chgnet_relax_steps_2000` and `e_form_per_atom_chgnet_relax_steps_500`, then it went wrong when I ran the file `analyze_chgnet.py`. Corresponding code can be seen as follows:

```py
e_form_2000 = "e_form_per_atom_chgnet_relax_steps_2000"
e_form_500 = "e_form_per_atom_chgnet_relax_steps_500"

min_e_diff = 0.1
# structures with smaller energy after longer relaxation need many steps
df_long = df_chgnet.query(f"{e_form_2000} - {e_form_500} < -{min_e_diff}")
# structures with larger energy after longer relaxation are problematic
df_bad = df_chgnet.query(f"{e_form_2000} - {e_form_500} > {min_e_diff}")
# both combined
df_diff = df_chgnet.query(f"abs({e_form_2000} - {e_form_500}) > {min_e_diff}")
```

**Comment by janosh** [comment, marked as answer]
that code is a bit outdated. the columns `e_form_(500|2000)` can be found in the `df_chgnet_v020` dataframe that's loaded right above the snippet you posted.

https://github.com/janosh/matbench-discovery/blob/31183303257ab0275f42ee000345b45036b077c1/models/chgnet/analyze_chgnet.py#L25-L35

feel free to submit a bug fix PR. that said, this analysis was only to check how much effect the max length of MLFF relaxation trajectory has on the ML final energy. answer: not that much. so you could just remove the failing code.

**Reply by Claudia-Hello** [reply]
thank you so much!
